registry.json

```diff
@@ -5403,5 +5403,20 @@
     "telegram": "caikejungle",
     "operatorName": "Carlos Souza",
     "github": "wowica"
+  },
+  {
+    "ticker": "1JPN",
+    "poolId": "54a7cf8a1e91ff0b72a3577c0d2f39d2d7ad1fca63ae0b9e4662053d",
+    "social": {
+      "website": "https://nihon-ichi-cardano.medium.com/",
+      "twitter": "sakakibara1JPN",
+      "facebook": "",
+      "discord": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "",
```

Comment:
You must provide the telegram username of an individual and not a group or channel